troposphere/validators.py

```diff
@@ -73,6 +73,14 @@ def s3_bucket_name(b):
         raise ValueError("%s is not a valid s3 bucket name" % b)
 
 
+def elb_name(b):
+    elb_name_re = compile(r'^[a-zA-Z0-9][a-zA-Z0-9\-]{0,30}[a-zA-Z0-9]{1}$')
```

Comment:
This worked! Nice call.  I've added it here: 9b55342afe7a2f14cb126f49441fb19a8d514470

I tested manually a bit on http://pythex.org/ just for context.